src/Authorization.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageWriter;
+
+use Keboola\Component\UserException;
+use Keboola\StorageApi\Client;
+
+class Authorization
+{
+    /**
+     * @var string
+     */
+    private $authorizedBucket;
+
+    public function __construct(Client $client)
```

Comment:
je to vůbec povolený, předávat v konstruktoru argumenty, který se neukládaj?

Reply:
co je na tom?

Reply:
nic, jen mě to zajímalo :-)